library/src/main/java/com/dd/CircularProgressButton.java

```diff
@@ -587,27 +600,27 @@ public void setStrokeColor(int color) {
         background.setStrokeColor(color);
     }
 
-    public String getIdleText() {
+    public CharSequence getIdleText() {
         return mIdleText;
     }
 
-    public String getCompleteText() {
+    public CharSequence getCompleteText() {
```

Comment:
Please help me. what's wrong with `String` ?

Reply:
`CharSequence`  is more flexible. 

E.g. `https://developer.android.com/reference/android/text/Spannable` implements `CharSequence` interface, so you can make text with different color, size, style, etc.